cmake/Dependencies.cmake

```diff
@@ -0,0 +1,51 @@
+find_package(Eigen3 REQUIRED NO_MODULE)
+
+if(VAMP_BUILD_PYTHON_BINDINGS)
+  find_package(Python 3.8
+    REQUIRED COMPONENTS Interpreter Development.Module
+    OPTIONAL_COMPONENTS Development.SABIModule)
+
+  CPMAddPackage("gh:wjakob/nanobind#358d452c314dbe8c07026d984ad8d5aa860f26fb")
+endif()
+
+CPMAddPackage("gh:kavrakilab/nigh#97130999440647c204e0265d05a997dbd8da4e70")
+if(nigh_ADDED)
```

Comment:
Do you actually need to check if it was added? The build should fail if it wasn't found?